mtg_card_identifier/CardScanner/CardScannerView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  CardScannerView.swift
+//  mtg_card_identifier
+//
+//  Created by mac on 8/16/23.
+//
+
+import SwiftUI
+import VisionKit
+import AVFoundation
+
+struct CardScannerView: View {
+    @State private var cameraAccessAuthorized = false
```

Comment:
This should probably be an enum with 3 states: allowed, not allowed, not determined.

Reply:
like this?  
 enum CameraAuthorized {
        case allowed
        case notAllowed
        case unDetermined
    }